editors/HexFiend/Portfile

```diff
@@ -5,12 +5,11 @@ PortGroup       xcode 1.0
 PortGroup       xcodeversion 1.0
 PortGroup       github 1.0
 
-github.setup    ridiculousfish HexFiend 2.9.0 v
-checksums       rmd160  ba4276249a262c1b808baafe3aa580ecfa75f224 \
-                sha256  158d398c7f723a1b1e622cabac7dd80b5556a64c8b649ab88e9989f45d2a39cc \
-                size    3474321
+github.setup    ridiculousfish HexFiend 2.13.1 v
+checksums       rmd160  7677158ec5d61351eae8086f0c37c04b9766c8c7 \
+                sha256  54a9eee7d2f3f25e8a7217fe89cdd6ee796a53e495e75e0e7dfdb5764d5ca34f \
+                size    3488533
 
-epoch           1
```

Comment:
This line needs to be restored. The `epoch` can never be decreased because anyone who has this port installed will not see this update.

Reply:
Restored the line.